addons/l10n_in/data/res_country_state_data.xml

```diff
@@ -1,13 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
+    <record id="state_in_oc" model="res.country.state">
+        <field name="name">Foreign Country</field>
+        <field name="code">IN_OC</field>
+        <field name="country_id" ref="base.in"/>
+        <field name="l10n_in_tin">96</field>
+    </record>
```

Comment:
This feels really weird.
I added a comment directly on the task.

If the goal is to give the `96` ad TIN code to any other state in another country, it could be done differently.